content/tutorials/marketplace.md

```diff
@@ -3,43 +3,19 @@ title: Set Up a Marketplace
 description:
 ---
 
-In Ocean, marketplaces and publishers are different roles. A common setup is for one organization to do both. We focus on that here.
+## About marketplace
 
-## The Steps
+Ocean Protocol's [marketplace](https://market.oceanprotocol.com/) provider a web interface for accessing assets published on the chain. By default, the assets are pulled from Aquarius hosted by Ocean Protocol. To extend the existing features of the marketplace, developers can fork the marketplace.
 
-1. Develop the first cut of the app.
-1. Prepare some initial data assets.
-1. Deploy to production.
+By doing so, developers can:
 
-## Develop a First Cut of the App
+- Change the name of the marketplace.
+- Implement their own branding, style.
+- Change the source of asset information.
+- Change fee address.
```

Comment:
"Change fee address.". This is about custom fees? I think we can rephrase this to "Customise the fees" or smth like that